codalab/worker/docker_image_manager.py

```diff
@@ -156,156 +126,157 @@ def get(self, image_spec):
             * READY if the image was downloaded successfully.
```

Comment:
Move the parts of this docstring about threads etc. to the parent class docstring.